Formula/sqlite_fdw.rb

```diff
@@ -0,0 +1,66 @@
+class SqliteFdw < Formula
+  desc "SQLite Foreign Data Wrapper for PostgreSQL"
+  homepage "https://github.com/pgspider/sqlite_fdw/"
+  url "https://github.com/pgspider/sqlite_fdw/archive/v1.3.0.tar.gz"
+  sha256 "61283b7f33b2a727af722fcfcdfe3cb7d6847e1f43e12a4b3554ab7190bfe9b3"
+  license "MIT"
+
+  depends_on "postgresql"
+  depends_on "sqlite"
+
+  def install
+    mkdir "stage"
+    system "make", "install", "USE_PGXS=1", "DESTDIR=#{buildpath}/stage"
```

Comment:
This way the next lines aren't needed.

```suggestion
    system "make", "install", "USE_PGXS=1", "DESTDIR=#{prefix}"
```

Reply:
Won't that install the files directly into `/usr/local`? If so, Homebrew won't know which files belong to this formula and which belong to the system, right? Or am I misunderstanding how Homebrew works?

As I said, I adapted the `wal2json` formula to make this one, so if this can be simplified, then the same change can be made to `wal2json`.

Reply:
`prefix` is `/usr/local/Cellar/sqlite_fdw/<version`, so that's where `make install` will place files. Homebrew then symlinks `#{prefix}/bin` into `#{HOMEBREW_PREFIX}/bin`, `#{prefix}/lib` into `#{HOMEBREW_PREFIX}/lib`, etc. (where `HOMEBREW_PREFIX` defaults to `/usr/local`).

Thus Homebrew always knows which files belong to the formula - because they live in the Cellar (which is where `prefix` points), and are only then symlinked into `HOMEBREW_PREFIX`.